spec/webauthn/attestation_statement/tpm_spec.rb

```diff
@@ -53,6 +54,9 @@
       let(:aik_certificate_subject) { "" }
       let(:aik_certificate_basic_constraints) { "CA:FALSE" }
       let(:aik_certificate_extended_key_usage) { WebAuthn::AttestationStatement::TPM::OID_TCG_KP_AIK_CERTIFICATE }
+      let(:aik_certificate_subject_alt_name) do
+        "DirName:/2.23.133.2.3=id:13/2.23.133.2.2=NPCT6xx/2.23.133.2.1=id:4E544300"
```

Comment:
This doesn't work, need to figure out if this can be done with `OpenSSL::X509::Name.parse` or craft the ASN1 data structures by hand... see https://www.trustedcomputinggroup.org/wp-content/uploads/Credential_Profile_EK_V2.0_R14_published.pdf paragraphs 3.1.2, 5, and appendix A.1